### PR TITLE
8.10 release

### PR DESCRIPTION
**Pull request by eabdullin1**

### CODE REVIEW

**Comment by eabdullin1** [comment]
@bennyvasquez @codyro @jonathanspw Please review, unfortunately, I can't add you to reviewers, because I can't edit fields

File: content/blog/2024-05-28-announcing-810-stable.md [hunk 1]
```diff
@@ -0,0 +1,100 @@
+---
+title: "General Availability of AlmaLinux 8.10 Stable!"
+type: blog
+author:
+ name: "Eduard Abdullin"
+ bio: "The Release and Automation Engineer."
```
Comment (codyro) [inline content/blog/2024-05-28-announcing-810-stable.md:6]: This sounds odd. It should be adjusted:

> Release and Automation Engineer

If you want to communicate you're the lead, perhaps `Principal Release and Automation Engineer` or `Lead Release and Automation Engineer` would be better.

File: content/blog/2024-05-28-announcing-810-stable.md [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+    * VMWare for x86_64 and AArch64
+    * Parallels for AArch64
+* [Raspberry Pi](https://wiki.almalinux.org/documentation/raspberry-pi.html)
+* [Windows Subsystem for Linux](https://wiki.almalinux.org/documentation/wsl.html) for x86_64 and AArch64
+
+## Release Notes and More Information
+
+AlmaLinux 8.10 brings updates to security and data protection, and improvements in web-console and system roles to automate operations and ensure consistency in intricate IT settings. The release continues to enhance system availability, reliability, and recovery processes, alongside improving virtual machine snapshot functions in hybrid cloud scenarios. New system roles have been introduced to streamline the creation and administration of logical volume manager (LVM) snapshots for better data backup and recovery processes. Performance, scalability, and reliability continue to be the focus of updates in the 8.10 version to aid developers in application development and management.
+
+You can read the full release notes for this version on the wiki: [AlmaLinux OS 8.10 Release Notes](https://wiki.almalinux.org/release-notes/8.10.html).
```
Comment (codyro) [inline content/blog/2024-05-28-announcing-810-stable.md:70]: 404 for now, https://wiki.almalinux.org/release-notes/8.10-beta.html needs to be renamed before this is published

(PR: https://github.com/AlmaLinux/wiki/pull/419/)

File: content/blog/2024-05-28-announcing-810-stable.md [hunk 3]
```diff
@@ -0,0 +1,100 @@
...
+    * **hpsa** - HP Smart Array Controller 
+    * **lpfc** - Emulex LightPulse Fibre Channel SCSI 
+    * **megaraid_sas** - Broadcom MegaRAID SAS 
+    * **mlx4_core** - Mellanox Gen2 and ConnectX-2 adapters
+    * **mpt3sas** - LSI MPT Fusion SAS 3.0 
+    * **mptsas** - Fusion MPT SAS Host 
+    * **qla2xxx** - QLogic Fibre Channel HBA 
+    * **qla4xxx** - QLogic iSCSI HBA 
+
+For a complete list of hardware support for which was added in this release, see the  [Extended hardware support](https://wiki.almalinux.org/release-notes/8.10.html#extended-hardware-support) section in the 8.10 release notes.
```
Comment (codyro) [inline content/blog/2024-05-28-announcing-810-stable.md:90]: Same as above, the wiki URI needs to be updated prior to publishing: https://wiki.almalinux.org/release-notes/8.10-beta.html#extended-hardware-support

**Review by codyro — CHANGES_REQUESTED** [review]
Looks good. One suggestion and a couple of notes (tl;dr merge wiki changes in beforet his is published). 

Good job @eabdullin1 :D!

File: content/blog/2024-05-28-announcing-810-stable.md (hunk 1 above)
Comment (eabdullin1) [reply content/blog/2024-05-28-announcing-810-stable.md]: Fixed

File: content/blog/2024-05-28-announcing-810-stable.md (hunk 2 above)
Comment (eabdullin1) [reply content/blog/2024-05-28-announcing-810-stable.md:70]: Wiki should be updated before merging it
In this case, we will be fine

File: content/blog/2024-05-28-announcing-810-stable.md (hunk 3 above)
Comment (eabdullin1) [reply content/blog/2024-05-28-announcing-810-stable.md:90]: Answer in previous comment)

File: content/blog/2024-05-28-announcing-810-stable.md [hunk 4]
```diff
@@ -0,0 +1,100 @@
...
+
+Torrents are available as well at:
+* [Intel/AMD (x86_64)](https://repo.almalinux.org/almalinux/8.10/isos/x86_64/AlmaLinux-8.10-x86_64.torrent)
+* [ARM64 (aarch64)](https://repo.almalinux.org/almalinux/8.10/isos/aarch64/AlmaLinux-8.10-aarch64.torrent)
+* [IBM PowerPC (ppc64le)](https://repo.almalinux.org/almalinux/8.10/isos/ppc64le/AlmaLinux-8.10-ppc64le.torrent)
+* [IBM Z (s390x)](https://repo.almalinux.org/almalinux/8.10/isos/s390x/AlmaLinux-8.10-s390x.torrent)
+
+## How we build
+
+Matching release and software versions with Red Hat Enterprise Linux(RHEL), AlmaLinux builds from the same sources as RHEL, promises complete compatibility with RHEL, and does so from freely available open source code. This makes it the only choice for anyone looking for a truly open source Enterprise Linux. If you are looking for a deeper undertanding of how AlmaLinux is built, watch this 23 minute video of our Lead Architect explaining in detail at [AlmaLinux Day: Germany](https://almalinux.org/almalinux-day-germany-2024/), earlier this year:
```
Comment (bennyvasquez) [inline content/blog/2024-05-28-announcing-810-stable.md:37]:
```suggestion
AlmaLinux matches release and software versions with Red Hat Enterprise Linux (RHEL) and builds from the same sources as RHEL, ensuring complete compatibility with RHEL, and does so from freely available open source code. If you are looking for a deeper understanding of how AlmaLinux is built, watch this 23-minute video of our Lead Architect explaining in detail at [AlmaLinux Day: Germany](https://almalinux.org/almalinux-day-germany-2024/), earlier this year:
```

**Review by bennyvasquez — APPROVED** [review]
One change. I adjusted the phrasing that we're using when pushing folks to the video a bit.

**Review by bennyvasquez — APPROVED** [review]

**Pull request merged**